benchmarks/bvh_driver/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ find_package(Threads REQUIRED)
 
 add_executable(ArborX_BoundingVolumeHierarchy.exe bvh_driver.cpp)
 target_link_libraries(ArborX_BoundingVolumeHierarchy.exe ArborX::ArborX benchmark::benchmark Boost::program_options Threads::Threads)
-target_include_directories(ArborX_BoundingVolumeHierarchy.exe PRIVATE ${POINT_CLOUDS_INCLUDE_DIR} ${UNIT_TESTS_INCLUDE_DIR})
+target_include_directories(ArborX_BoundingVolumeHierarchy.exe PRIVATE ${POINT_CLOUDS_INCLUDE_DIR} ${UNIT_TESTS_INCLUDE_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I would use double quotes for headers in that directory instead but I am not blocking.

Reply:
Followed the suggestion.